app/containers/ViewQuestion/Bounty.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import { APP_FONT } from 'style-constants';
```

Comment:
import elements in groups. use some logic. for example, constants separately, libraries separately, components separately, actions separately etc.